leaderboard.html

```diff
@@ -51,10 +51,10 @@
         z-index: 100;
         font-size: 18px;
       ">
-        <a href="#leaderboard">Leaderboard</a>
-        <a href="#api-explorer">Try it Out!</a>
+        <a href="index.html">Home</a>
         <a href="blogs/8_berkeley_function_calling_leaderboard.html">Blog</a>
-        <a href="index.html">Gorilla</a>
+        <a href="#api-explorer">Try it Out!</a>
+        <a href="#leaderboard">Leaderboard</a>
```

Comment:
Can the last botton be api zoo index, rather than leaderboard, since we are currently in leaderboard (to be consistent with home page).
<img width="1098" alt="image" src="https://github.com/ShishirPatil/gorilla/assets/55744150/5e4b58c0-369a-426f-a0c5-ec7dae1991e5">

Reply:
We discussed this before. This is the leaderboard page, we want leaderboard here. It doesn't make sense to redirect to API zoo index.

Reply:
Same idea as why the blog in leaderboard redirect to blog 8, instead of the blog landing page.

Reply:
Sure, can you also modify the navbar `Gorilla` to `Home` for https://huanzhimao.com/gorilla/blogs/8_berkeley_function_calling_leaderboard.html for consistency?

Reply:
Sure. This I agree. We should let the blogs have consistent nav bar.